examples/06_timers.rs

```diff
@@ -0,0 +1,48 @@
+// Example 6: The Window
+// Use timers to know when to draw and to have a consistent update cycle.
+use quicksilver::{
+    geom::{Rectangle, Vector},
+    graphics::{Color, Graphics},
+    lifecycle::{run, EventStream, Settings, Window},
+    Result, Timer,
+};
+
+fn main() {
+    run(
+        Settings {
+            size: Vector::new(800.0, 600.0).into(),
+            title: "Square Example",
+            ..Settings::default()
+        },
+        app,
+    );
+}
+
+async fn app(window: Window, mut gfx: Graphics, mut events: EventStream) -> Result<()> {
+    // Clear the screen to a blank, white color
+    gfx.clear(Color::WHITE);
+
+    //Here we make 2 kinds of timers.
```

Comment:
This looks great, some sort of timer to help with implementing proper frame rates is definitely something that quicksilver-alpha needs. I have just one small nitpick: everywhere else in the code base has a space after a comment `//`, but a lot of the comments in this PR do not. Other than that the PR looks great, I especially like the `while update_timer.tick()` syntax.

Reply:
Most (if not all) of the syntax comes from Ryan, so you have him to thank for that :).

Anyway, I fixed the comment.